HabitTracker.harris-andy/Program.cs

```diff
@@ -0,0 +1,592 @@
+﻿using System.Data;
+using System.Globalization;
+using Microsoft.Data.Sqlite;
+
+string connectionString = @"Data Source=HabitTracker.db";
+
+using (SqliteConnection connection = new SqliteConnection(connectionString))

```

Comment:
Remember that you can use the simpler "Using Declaration" syntax to save an indentation level against the "Using Statement" syntax:

(Note I only put this is a Microsoft push it in their style guide which I adhere to (try), but it is user preference)

❌ 
```
// Using Statement
using (var connection = new SqliteConnection(connectionString))
{
    // ...
}
```

✔️ 
```
// Using Declaration
using var connection = new SqliteConnection(connectionString);
// ...
```